workspace-manager-clienttests/src/main/resources/configs/resiliency/DeleteInitialPods.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "DeleteInitialPods",
+  "description": "Run two Service Status tests concurrently with an artificial delay, while incrementally deleting pods.",
```

Comment:
You don't need two `testScripts` to do this. A single one, with `numberOfUserJourneyThreadsToRun`=2 and `userJourneyThreadPoolSize`=2 , should be equivalent.